src/app/components/Labels/DeleteLabelButton.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { c } from 'ttag';
+import {
+    ConfirmModal,
+    Alert,
+    Button,
+    useModal,
+    useApiWithoutResult,
+    useEventManager,
+    useNotifications
+} from 'react-components';
+import { deleteLabel } from 'proton-shared/lib/api/labels';
+import { noop } from 'proton-shared/lib/helpers/function';
+import { LABEL_TYPES } from 'proton-shared/lib/constants';
+
+function DeleteLabelButton({ label, onRemove }) {
+    const { call } = useEventManager();
+    const { createNotification } = useNotifications();
+    const { request, loading } = useApiWithoutResult(deleteLabel);
+    const { isOpen: isOpenConfirmModal, open: openConfirmModal, close: closeConfirmModal } = useModal();
```

Comment:
You don't need to rename variables here, there is no conflict in this component.